python/ray/serve/tests/test_deployment_state.py

```diff
@@ -2583,6 +2582,7 @@ def test_default_value(self):
         assert actor_replica.health_check_period_s == DEFAULT_HEALTH_CHECK_PERIOD_S
         assert actor_replica.health_check_timeout_s == DEFAULT_HEALTH_CHECK_TIMEOUT_S
 
+    @patch("ray.util.get_placement_group", lambda *args: "test_pg")
     def test_recover(self):
```

Comment:
this test looks useless, i'd just delete it

Reply:
agrred, done!